docs/events/focus.md

```diff
@@ -12,3 +12,9 @@ _No other attributes_
 ## Code
 
 ::: textual.events.Focus
+
+## See also
+
+- [Blur](blur.md)
+- [DescendantFocus](descendant_focus.md)
+- [DescendantBlur](descendant_blur.md)
```

Comment:
tsk tsk
```suggestion
- [DescendantBlur](descendant_blur.md)
- [DescendantFocus](descendant_focus.md)
```

Reply:
Just checking that you're reviewing properly. :-P